docs/config.md

```diff
@@ -547,6 +547,10 @@ merge-tools.vimdiff.merge-args = ["-f", "-d", "$output", "-M",
     "-c", "set write"]
 merge-tools.vimdiff.program = "vim"
 merge-tools.vimdiff.merge-tool-edits-conflict-markers = true    # See below for an explanation
+
+# For IntelliJ IDEA or other JetBrains products on macOS
+merge-tools.idea.program = "/Applications/IntelliJ IDEA.app/Contents/MacOS/idea"
```

Comment:
We could add a note that for Linux, the launcher script must be installed and that this value must be pointed to it.

https://www.jetbrains.com/help/idea/working-with-the-ide-features-from-command-line.html